src/test/java/cyclops/data/LazySeqTest.java

```diff
@@ -79,6 +81,76 @@ public void testLazy(){
 
 
   }
+  /**
+   * Test no maps are called before fetching values and map function only invoked once for each input.
+   *
+   * @throws Exception
+   */
+  // TODO(johnmcclean): Stop ignoring this test
+  @Ignore
+  @Test
+  public void testLazy2() throws Exception {
```

Comment:
If you use a method name which indicates what is being tested, then you can avoid the java doc comment, also easy to understand the  intent of the test from the ngunit test report. eg `testNoMapCalledBeforeFetchingAndOnlyOnceForEachInput`

Reply:
Done